### ISSUE TITLE
Issue in deleting dependent snippet

### ISSUE THREAD

**Issue by neelasha23** [ploomber/jupysql#717]
If save two snippets `A` and `B`, where `B` depends on `A`, and then `force-delete` `A`, then it is not possible to delete `B`. PFB error:

<img width="1108" alt="Screenshot 2023-07-07 at 4 02 58 PM" src="https://github.com/ploomber/jupysql/assets/29096135/9d5cdf58-480e-4021-b579-04d66aaa16a6">

**Comment by edublancas** [comment]
@neelasha23 is this still an issue? I don't see any PRs

**Comment by neelasha23** [comment]
yes I started working on the jupysql-plugin issues so didn't pick this up. I'll start on it after the CTE bracket bug fix?

**Comment by edublancas** [comment]
sure, please work on the plugin issues first
